packages/axios-client-adapter/src/httpClient.ts

```diff
@@ -213,6 +219,21 @@ export class HttpClient {
     }
   }
 
+  public setProxyAgent(axiosRequest: AxiosRequestConfig): void {
```

Comment:
This function should be private, and only should be used within this class. I couldn't find any usages of this function here^